LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/CupsTwinInfo.cs

```diff
@@ -51,5 +57,17 @@ internal sealed record CupsTwinInfo
         [JsonPropertyName("tcCredentialUrl")]
         [JsonProperty("tcCredentialUrl")]
         public string TcCredentialUrl { get; }
+
+        [JsonPropertyName("package")]
+        [JsonProperty("package")]
+        public string Package { get; }
+
+        [JsonPropertyName("fwKeyChecksum")]
+        [JsonProperty("fwKeyChecksum")]
+        public int FwKeyChecksum { get; }
```

Comment:
Is the checksum a _signed_ integer?

Reply:
No, it's an unsigned one. Thanks for catching it